rllib/core/rl_trainer/tests/test_trainer_runner.py

```diff
@@ -1,6 +1,5 @@
 import gymnasium as gym
 import unittest
-import pytest
 
 import tensorflow as tf
```

Comment:
Don't do this here man, I did this in the clean up PR which is gonna get merged in a bit.

Reply:
Let's wait for that to get merged and then combine the two?